atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/ClientLockWatchSnapshot.java

```diff
@@ -98,12 +98,12 @@ Optional<LockWatchVersion> getSnapshotVersion() {
         return snapshotVersion;
     }
 
-    @VisibleForTesting
-    ClientLockWatchSnapshotState getStateForTesting() {
+    @Unsafe
+    ClientLockWatchSnapshotState getStateForDiagnostics() {
         return ImmutableClientLockWatchSnapshotState.builder()
                 .snapshotVersion(snapshotVersion)
-                .locked(locked)
-                .watches(watches)
+                .locked(new HashSet<>(locked))
+                .watches(new HashSet<>(watches))
                 .build();
```

Comment:
optional: Any reason we didn't use ImmutableSet.copyOf(...) here? I don't imagine these get mutated

Reply:
no reason really, just was the first thing I reached out for, will change